locales/fr.js

```diff
@@ -10,7 +10,7 @@
   "Bump done": "Bump effectué ! ",
   "Check it on DISBOARD": "Allez vérifier ça sur DISBOARD ",
   "Please give me `Create Instant Invite` permission on this channel": "Veuillez me donner la permission `Créer une invitation` dans ce salon",
-  "Please give me `Manage Channels` permission in order to receive invite change event": "Please give me `Manage Channels` permission in order to receive invite change event",
+  "Please give me `Manage Channels` permission in order to receive invite change event": "Veuillez me donner la permission `Gérer les salons` pour que je puisse modifier l'invitation via la commande appropriée",
```

Comment:
"via la commande appropriée" it's not a specific command per se, it's just when you do `!d invite` the bot has to know whether the invite is changed or not

Reply:
Could be changed to ```Veuillez me donner la permission `Gérer les salons` afin que je puisse gérer le changement d'invitation```?

Reply:
didn't realize i merged it anyway, fixed it in https://github.com/austinhuang0131/disboard-i18n/commit/4864c3213ec2512f36fa7c4439705fd4bb19cd49